app/views/provider_mailer/unconditional_offer_accepted.text.erb

```diff
@@ -3,7 +3,7 @@ Dear <%= @provider_user.full_name || 'colleague' %>
 
 # Offer accepted
 
-<%= @application_choice.application_form.full_name %> has accepted your offer to study <%= @application_choice.current_course.name_and_code %>.
+<%= @application_choice.application_form.full_name %> has accepted your offer to study <%= @application_choice.current_option.course.name_and_code %>.
```

Comment:
Why can't we have `current_course` here and above?

Reply:
The main issue I've had with the mailers is the previews, which use `build_stubbed` and also reference `current_option` for other things. I could have also specified `current_course` for the stubbed models but it felt like setting current_option is safer, as it can also be used for other things. Specifying `current_course` and `current_option` should work, but my main priority was to get all tests to pass without spending too much time on single cases.